src/gmail.js

```diff
@@ -1995,24 +2002,41 @@ var Gmail = function(localJQuery) {
                 var events = false;
                 if (this.xhrParams) {
                     this.xhrParams.body = body;
+
+                    // restore original values of xhrParams, if they were altered by upstream instances of gmail.js
+                    if (typeof this.xhrParams.url !== 'string') {
+                        if (
+                            this
```

Comment:
I appreciate the “safety first” approach in this critical code-path, but can “this” ever be null here?

Reply:
Yes, you're right. Will remove.